rskj-core/src/main/java/co/rsk/peg/utils/BridgeEventLoggerImpl.java

```diff
@@ -93,6 +96,14 @@ public void logCommitFederation(Block executionBlock, Federation oldFederation,
         this.logs.add(new LogInfo(BRIDGE_CONTRACT_ADDRESS, topics, data));
     }
 
+    public void logLockBtc(BtcTransaction btcTx, Address senderBtcAddress, RskAddress RskReceiver, Coin Amount) {
+        List<DataWord> topics = Collections.singletonList(Bridge.LOCK_BTC_TOPIC);
+        byte[] data = RLP.encodeList(RLP.encodeString(btcTx.getHashAsString()), RLP.encodeString(senderBtcAddress.toString()),
```

Comment:
You shouldn't be logging the btcTx. Additionally, use `toBase58` instead of `toString` when serializing the btc address